webpack.config.js

```diff
@@ -75,6 +75,30 @@ const babelLoader = {
 	},
 };
 
+/**
+ * Converts @wordpress require into window reference
+ *
+ * Note this isn't the same as camelCase becuase of the
+ * way that numbers don't trigger the capitalized next letter
+ *
+ * @example
+ * wordpressRequire( '@wordpress/api-fetch' ) = 'wp.apiFetch'
+ * wordpressRequire( '@wordpress/i18n' ) = 'wp.i18n'
+ *
+ * @param {string} request import name
+ * @return {string} global variable reference for import
+ */
+const wordpressRequire = request => {
+	const [ , /* @wordpress */ name ] = request.split( '/' );
```

Comment:
Should `/* @wordpress */` be listed before the comma?

Reply:
yes but `prettier` moves it back - a long-standing bug sadly :(

Reply:
adjusted to it's clearer